apps/framework-cli/src/utilities/capture.rs

```diff
@@ -57,39 +59,38 @@ macro_rules! capture {
         use chrono::Utc;
         use reqwest::Client;
         use serde_json::json;
+        use std::env;
+        use std::time::Duration;
         use uuid::Uuid;
 
-        #[allow(unused)]
-        let event = json!(MooseActivity {
-            id: Uuid::new_v4(),
-            project: $project_name,
-            activity_type: $activity_type,
-            sequence_id: $sequence_id,
-            timestamp: Utc::now(),
-            cli_version: constants::CLI_VERSION.to_string(),
-        });
-        let remote_url = {
-            let guard = PROJECT.lock().unwrap();
-            guard.instrumentation_config.url().clone()
-        };
-
-        // Sending this data can fail for a variety of reasons, so we don't want to
-        // block user & no need to handle the result
-        tokio::spawn(async move {
```

Comment:
I originally wanted to send the data in a spawned thread, but the main thread can actually finish faster and kill the spawned thread. So I just added a quick timeout to the request. Let me know if there's a better way we can spray-and-pray (a.ka. send the data without blocking user & without checking on the result).